compiler/src/main/java/org/qbicc/type/BooleanType.java

```diff
@@ -50,4 +50,8 @@ public StringBuilder toString(final StringBuilder b) {
     public StringBuilder toFriendlyString(final StringBuilder b) {
         return b.append("bool");
     }
+
+    public PrimitiveType asJavaType() {
```

Comment:
`asJavaType` is a bit at odds with the class name `PrimitiveType` IMO; they should be more consistent.

Reply:
Renamed `asJavaType` to `asPrimitive`.